src/jest-types.ts

```diff
@@ -34,6 +34,7 @@ export type JestConfig = Partial<FullJestConfig>;
 
 export interface FullJestConfig {
   automock: boolean;
+  babelConfig: object;
```

Comment:
is this field needed here?

Reply:
`yarn run tslint` will still pass.  For contributors with type-safety checks enabled, it's useful to have accurate types, though.

Reply:
I know that `babelConfig` is part of `TsJestConfig`. Does the Jest config also allow setting `babelConfig`?

Reply:
Ahhh, yes, this change makes no sense!  I've removed it.  Thank you :)